src/test/java/org/folio/rest/impl/OrdersResourceImplTest.java

```diff
@@ -207,6 +206,11 @@ protected Router defineRoutes() {
       router.route().handler(BodyHandler.create());
       router.route(HttpMethod.POST, "/purchase_order").handler(this::handlePostPurchaseOrder);
       router.route(HttpMethod.POST, "/po_line").handler(this::handlePostPOLine);
+      router.route(HttpMethod.POST, "/details").handler(this::handlePostAssignId);
```

Comment:
Will these router calls exercise all the helper methods that were defined?

Reply:
When we run these tests, they makes calls to the /orders API, providing an X-Okapi-Url header pointing to localhost, where we have this mock server running.  The orders code will run and make calls to ${X-Okapi-Url}/cost, ${X-Okapi-Url}/details, etc.  For now the the mock server just returns enough dummy data to get by (essentially the request body with a populated id/UUID).  Eventually, in the interest of completeness I/we should add some negative test cases here.  

If it makes you feel any better, parts of the exceptional path(s) involved here are shared, e.g. verifyAndExtractBody(...)/handleError(...) and are already covered, but we can and should do better.

Reply:
I'll create a JIRA ticket for improving the unit tests, including additional negative test cases

Reply:
Sounds good!

Reply:
FYI https://issues.folio.org/browse/MODORDERS-34